.github/workflows/build-release-dev.yml

```diff
@@ -0,0 +1,63 @@
+name: build-and-release-dev
+
+on:
+  push:
+    branches-ignore:
+      - 'main'
+
+#on: workflow_dispatch
+
+
+jobs:
+  release:
+    runs-on: self-hosted
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'adopt'
+
+      - name: Set Gradle
+        uses: gradle/gradle-build-action@v2
+        with:
+          gradle-version: 7.4.2
+
+      - name: Set env vars from AWS params
+        uses: cmsgov/ab2d-bcda-dpc-platform/actions/aws-params-env-action@main
+        env:
+          AWS_REGION: ${{ vars.AWS_REGION }}
+        with:
+          params: |
+            ARTIFACTORY_URL=/artifactory/url
+            ARTIFACTORY_USER=/artifactory/user
+            ARTIFACTORY_PASSWORD=/artifactory/password
+            SONAR_HOST_URL=/sonarqube/url
+            SONAR_TOKEN=/sonarqube/token
+
+      - name: Build Libraries
+        run: |
+          gradle -b build.gradle bootJar -Dset.root.project.build.filename=true
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v3
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::${{ vars.DEV_ACCOUNT_NUMBER }}:role/delegatedadmin/developer/github-actions-runner-role
+
+      - name: Build Docker Image
+        env:
+          ACCOUNT_NUMBER: ${{ vars.DEV_ACCOUNT_NUMBER }}
+          ECR_REPO_ENV: "ab2d-dev-services"
+        run: ./scripts/github-actions-build-docker-image.sh
+
+      - name: release to ecs 
+        env:
+          ACCOUNT_NUMBER: ${{ vars.DEV_ACCOUNT_NUMBER }}
+          ECR_REPO_ENV: "ab2d-dev-services"
+          DEPLOYMENT_ENV: "ab2d-dev"
```

Comment:
It looks like the only variable you need to pass to the github-actions-release script is DEPLOYMENT_ENV.